pkg/services/store/service.go

```diff
@@ -18,7 +23,7 @@ import (
 var grafanaStorageLogger = log.New("grafanaStorageLogger")
 
 const RootPublicStatic = "public-static"
-
+const MAX_UPLOAD_SIZE = 1024 * 1024 // 1MB
```

Comment:
Could this be customized?

Reply:
yeah good call - it should be customizable within filestorage I believe.